spec/student_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'spec_helper'
+
+describe Student do
+  student = Student.new('C', 30, 30, name: 'David')
+
+  it 'should be an instance of Student' do
+    expect(student).to be_an_instance_of(Student)
+  end
+  it 'should have a name' do
+    expect(student.name).to eql('David')
+  end
+  it 'should have rentals' do
+    expect(student.rentals).to eql([])
+  end
+  it 'should have an age' do
+    expect(student.age).to eql(30)
+  end
+  it 'shoild have a method play_hooky' do
+    expect(student).to respond_to(:play_hooky)
+  end
+end
```

Comment:
- [ ] Kindly make sure to add unit tests for all the methods of the: `Student class` ✍🏻 Cureent you're missing the `classroom` method tests. 👇🏻  


```ruby
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
```